thriftbp/server_transport.go

```diff
@@ -0,0 +1,46 @@
+package thriftbp
+
+import (
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/reddit/baseplate.go/metricsbp"
+)
+
+// TODO(marco.ferrer):1/4/22 Replace with metric name that conforms to bp conventions
+const meterNameTransportConnCounter = "thrift.conn.count"
```

Comment:
Not sure what the convention is for naming metrics in baseplate so I left this placeholder to update

Reply:
Baseplate specific metrics go under `baseplate` but in this case it might be similar to Thrift client pool stats which we put under `pool-active-connections` and `pool-allocated-clients`. What I would say though is to name it just `thrift.connections`. The count part is an aggregation option and would come up anyway.